### PR TITLE
Treeview

### PR DESCRIPTION
**Pull request by gheorghitamutu**

### CODE REVIEW

File: AppCUI/src/Controls/TreeView.cpp [hunk 1]
```diff
@@ -436,14 +455,29 @@ bool TreeView::OnKeyEvent(Input::Key keyCode, char16 character)
...
+                        TreeViewItem tvi{ this, handle };
+                        cc->SetSelectedItemHandle(tvi);
                         return true;
                     }
                 }
             }
         }
         break;
 
+    case Key::Enter:
```
Comment (gdt050579) [inline AppCUI/src/Controls/TreeView.cpp:467]: ENTER sau SPACE ar trebui sa aiba comportament similar

File: AppCUI/include/AppCUI.hpp [hunk 2]
```diff
@@ -2981,6 +2981,8 @@ namespace Controls
         using OnLoseFocusHandler = void (*)(Reference<Controls::Control> control);
         using OnStartHandler     = void (*)(Reference<Controls::Control> control);
         using OnTreeItemToggleHandler    = void (*)(TreeViewItem& item);
+        using OnTreeItemSelectedHandler  = void (*)(TreeViewItem& item);
```
Comment (gdt050579) [inline AppCUI/include/AppCUI.hpp:2984]: Schimba semnayura sa contina si referinta la TreeView.
De genu:
`using OnTreeItemToggleHandler = void (*)(Referemce<TreeView>, TreeViewItem& item);`

File: AppCUI/src/Controls/TreeView.cpp (hunk 1 above)
Comment (gheorghitamutu) [reply AppCUI/src/Controls/TreeView.cpp:467]: ok

File: AppCUI/include/AppCUI.hpp (hunk 2 above)
Comment (gheorghitamutu) [reply AppCUI/include/AppCUI.hpp:2984]: ok

**Review by gdt050579 — APPROVED** [review]
ok

**Pull request merged**